app/_data/kong_versions.yml

```diff
@@ -387,6 +401,10 @@
   release: "2.2.x"
   version: "2.2"
   edition: "getting-started-guide"
+-
+  release: "2.3.x"
+  version: "2.3"
+  edition: "getting-started-guide"
```

Comment:
Looks like this is breaking the build, because the /getting-started-guide folder and nav weren't cloned to 2.3.x. Can you either remove this entry for now, or clone the folder and nav to 2.3.x?